nilearn/datasets/atlas.py

```diff
@@ -782,3 +782,83 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_allen_rsn_tmap_75_2011(data_dir=None, url=None, resume=True, verbose=1):
```

Comment:
I just saw that the fetch_atlas_smith_2009 is also for a RSN template. So your suggestion sounds good.